services/metadata/backend/data/woposs.json

```diff
@@ -0,0 +1,492 @@
+            "Dell’Oro, Francesca (2022). “Kagan, O. (2020). The Semantics of Case. Key Topics in Semantics and Pragmatics. Cambridge: Cambridge University Press. DOI:10.1017/9781108236867”. LINGUIST List, 33.2004. Jun 15, 2022. https://linguistlist.org/issues/33.2004.",
+            "Dell’Oro, Francesca; Rossari, Corinne (ed.). “Theoretical issues in the light of the interaction between quantitative and qualitative data: recent approaches and tendencies”. Thematic issue of TRANEL (Travaux neuchâtelois de linguistique).",
+            "Bermúdez Sabel, Helena; Dell’Oro, Francesca; Montrichard, Cyrielle; Rossari, Corinne (2022). “Setting up bilingual comparable corpora with non-contemporary languages”. Proceedings of the LREC 2022 15th Workshop on building and using comparable corpora . Edited by R. Rapp, P. Zweigenhaum, S. Sharoff. Paris: European Language Resources Association (ELRA), 52–60.",
+            "Dell’Oro, Francesca; Bermúdez Sabel, Helena; Marongiu, Paola (2020). “Implemented to be shared: the WoPoss annotation of semantic modality in a Latin diachronic corpus”. Sharing the Experience: Workflows for the Digital Humanities. Proceedings of the DARIAH-CH Workshop 2019. December 5-6, 2019. Neuchâtel, Switzerland.",
+            "McGillivray, Barbara; Kondakova, Daria; Bermúdez Sabel, Helena (2022). Code for analysing the semantic annotation of Latin data from SemEval 2020 task 1. Zenodo. DOI: 10.5281/zenodo.6325749.",
+            "Dell’Oro, Francesca; Rimaz, Loris; Bermúdez Sabel, Helena; Marongiu, Paola (2021). Pygmalion-colour 1.0. A tool to draw interactive and diachronic maps . WoPoss. A World of Possibilities. Swiss National Science Foundation.",
+            "Dell’Oro, Francesca; Rimaz, Loris; Bermúdez Sabel, Helena; Marongiu, Paola (2020). Pygmalion-simple 1.0. A tool to draw interactive and diachronic maps of semantics . WoPoss. A World of Possibilities. Swiss National Science Foundation.",
+            "Dell’Oro, Francesca; Rimaz, Loris; Bermúdez Sabel, Helena; Marongiu, Paola (2020). Pygmalion-modal 1.0. A tool to draw interactive and diachronic maps of modality . WoPoss. A World of Possibilities. Swiss National Science Foundation.",
+            "Dell'Oro, Francesca; Bermúdez Sabel, Helena; Marongiu, Paola & Reymond, Séverine (2022). Sample of the WoPoss corpus (28th July 2022). Zenodo. Swiss National Science Foundation. DOI: 10.5281/zenodo.6925274.",
+            "Marongiu, Paola & Dell'Oro, Francesca. Interactive diachronic maps of the WoPoss modal markers. Schweizerischer Nationalfonds zur Förderung der wissenschaftlichen Forschung - SNF, Historical linguistics – Institut des sciences du langage - ISLa. Distributed by SWISSUbase, Lausanne, 2022. DOI: 10.48656/qk1e-kk67.",
+            "Dell’Oro, Francesca (2021). “Annotating (ancient) texts and learning (ancient) languages: a possible connection”. LexTech Institute. October 5, 2021."
```

Comment:
Not really our problem, but the quotation marks here are a bit all over the place

Reply:
yeah, I know